node/src/chain_spec/battery_station.rs

```diff
@@ -1,37 +1,129 @@
-#[cfg(not(feature = "without-sudo"))]
-use crate::chain_spec::root_key_staging_testnet;
-use crate::chain_spec::{
-    additional_chain_spec_staging_testnet, endowed_accounts_staging_testnet, generic_genesis,
-    telemetry_endpoints, token_properties, zeitgeist_wasm, ChainSpec,
-};
+#![cfg(feature = "with-battery-station-runtime")]
+
+use super::{AdditionalChainSpec, EndowedAccountWithBalance};
+use crate::chain_spec::{generate_generic_genesis_function, telemetry_endpoints, token_properties};
+use battery_station_runtime::parameters::SS58Prefix;
+use hex_literal::hex;
 use sc_service::ChainType;
+use sp_core::crypto::UncheckedInto;
+use zeitgeist_primitives::{
+    constants::{
+        ztg::{LIQUIDITY_MINING, LIQUIDITY_MINING_PTD},
+        BASE,
+    },
+    types::AccountId,
+};
+
+#[cfg(feature = "parachain")]
+use {
+    super::{Extensions, DEFAULT_COLLATOR_INFLATION_INFO},
+    battery_station_runtime::{CollatorDeposit, EligibilityValue, PolkadotXcmConfig},
+};
+
+cfg_if::cfg_if! {
+    if #[cfg(feature = "parachain")] {
+        pub(super) const DEFAULT_STAKING_AMOUNT_BATTERY_STATION: u128 = 2_000 * BASE;
+        const DEFAULT_COLLATOR_BALANCE_BATTERY_STATION: Option<u128> =
+            DEFAULT_STAKING_AMOUNT_BATTERY_STATION.checked_add(CollatorDeposit::get());
+        const DEFAULT_INITIAL_CROWDLOAN_FUNDS_BATTERY_STATION: u128 = 100 * BASE;
```

Comment:
When launching a staging-network (like `battery-station-staging`), which is usually used to spawn new networks and generate a chainspec for them, those constants set the amount that is staked per collator and the initial balance of the collator. It is correct because it is sufficient to stake and map an author key. However, I don't see where the author key mapping fee is considered, but it's covered by the additional `CollatorDeposit` that's added to the collator balance.

Reply:
But what role does the `DEFAULT_INITIAL_CROWDLOAN_FUNDS_BATTERY_STATION` play?

Reply:
It initializes the crowload genesis funding. AFAIK this is the amount of ZTG initially allocated for bidding for parachain slots on the relay chain. The amount of relay tokens, let's say KSM, that are available to be crowdfunded is determined by a ratio, let's say 80 ZTG / 1 KSM. I don't think there is any communication with the relay chain involved, instead it is proposed that the governance mechanism supplies the final reward vector. The tokens are also linearly vested (can be any monotonously rising function).

I think since both runtimes are separate now, we can probably remove that from battery station, because no parachain slot bidding is necessary there. Maybe there are other use-cases for that pallet that I don't see.
AFAIK we didn't even use it for the parachain bidding on Kusama, although initially planned. It would make sense though to leave it for the next bidding, because it would ease executing the crowdloan through governance.

Reply:
Sure, I don't mind having this. Was just curious why this value was set on a testnet. Thanks for elaborating!

Reply:
That's actually a great catch, it's something that we don't really need on the testnet. The first use-case in terms of different pallets for multiple runtimes besides *sudo* :tada:

Issue https://github.com/zeitgeistpm/zeitgeist/issues/745 created.